pkg/applicationmanager/controllers/applicationrestore.go

```diff
@@ -361,6 +361,7 @@ func (a *ApplicationRestoreController) handle(ctx context.Context, restore *stor
 		}
 	}
 
+	logrus.Infof("StorageClassMapping restore spec storage class mapping is: %v", restore.Spec.StorageClassMapping)
```

Comment:
Why is this log added in this pR?  Merge conflicts?

Reply:
There was a case when while restoring, storage class was not being taken in destination cluster by default, had used this log to debug and find out the issue, I forgot to remove the all the log messages that I had used while developing. Will be removing these. Thanks for pointing them out!